util.ts

```diff
@@ -1,31 +1,29 @@
 import { state } from "membrane";
 import fetch from "node-fetch";
 
+const api_url = "https://api.twitter.com"
+
 export async function api(
   method: string,
   path: string,
   query?: any,
   body?: string
-) {
-  if (!state.token) {
-    throw new Error(
-      "You must first invoke the configure action with an API token"
-    );
-  }
```

Comment:
Any reason to remove this? It's good to see this error when a user forgets to invoke `configure`